integration-tests/pom.xml

```diff
@@ -148,6 +148,8 @@
         <module>logging-json</module>
         <module>jaxp</module>
         <module>mailer</module>
+        <module>logging-min-level-unset</module>
+        <module>logging-min-level-set</module>
```

Comment:
If you want these modules to be tested on CI in native, they need to be added there: https://github.com/quarkusio/quarkus/blob/master/.github/native-tests.json .